### PR TITLE
Backend devbranch iqr and sd

### PR DESCRIPTION
**Pull request by sturdyplum**

### CODE REVIEW

**Comment by sturdyplum** [comment]
I tried to make tests for the new functions but I was getting an issue with importing functions from logic so I wasn't able to.

**Comment by calveym** [comment]
Looks good to me, I'm going to do a bit of testing and then I'll merge.

**Comment by calveym** [comment]
Found a couple bugs, mostly in reused code for averaging. Run the tests and have a look, bit weird. We might have to refactor some of the code for averages, it's a bit messy now anyway.

File: logic/functions.py [hunk 1]
```diff
@@ -7,13 +8,16 @@ class Processing:
...
+    :standardDeviation: defines standard deviation for inout data.
+
     Note: JSON imports INCOMPLETE
     """
     processeddata = {}
 
     def average(self, *args):
         """Simple average calculator"""
-        total = sum(args)
+        total = sum(args)  # Error starts here, and goes up the stack.
```
Comment (calveym) [inline logic/functions.py:20]: Main error starts here

**Pull request merged**